### PR TITLE
chore: improve Operator description

### PR DESCRIPTION
**Pull request by ivanstanev**
Enrich the Operator descriptions and text fields for a better UI/installation experience.
Also, explicitly list the prerequisites (setting up the snyk-monitor secret) in the description of the Operator.

- [ ] Tests written and linted [ℹ︎](https://github.com/snyk/general/wiki/Tests)
- [ ] Documentation written [ℹ︎](https://github.com/snyk/general/wiki/Documentation)
- [x] Commit history is tidy [ℹ︎](https://github.com/snyk/general/wiki/Git)

### More information

- [Jira ticketRUN-733](https://snyksec.atlassian.net/browse/RUN-733)
- [PR to OperatorHub](https://github.com/operator-framework/community-operators/pull/1296)

### CODE REVIEW

File: snyk-operator/deploy/olm-catalog/snyk-operator/0.0.0/charts.helm.k8s.io_snykmonitors_crd.yaml [hunk 1]
```diff
@@ -1,24 +0,0 @@
-apiVersion: apiextensions.k8s.io/v1beta1
```
Comment (talik077) [inline snyk-operator/deploy/olm-catalog/snyk-operator/0.0.0/charts.helm.k8s.io_snykmonitors_crd.yaml]: if the intention is to rename the file can we use `git mv` to preserve the git history?

File: snyk-operator/deploy/olm-catalog/snyk-operator/0.0.0/snykmonitors.charts.helm.k8s.io.crd.yaml [hunk 2]
```diff
@@ -10,12 +10,42 @@ spec:
     plural: snykmonitors
     singular: snykmonitor
   scope: Namespaced
-  subresources:
-    status: {}
   validation:
     openAPIV3Schema:
       type: object
-      x-kubernetes-preserve-unknown-fields: true
+      properties:
```
Comment (ivanstanev) [inline snyk-operator/deploy/olm-catalog/snyk-operator/0.0.0/snykmonitors.charts.helm.k8s.io.crd.yaml]: This actually now adds validation for the CRD 😵 Would you prefer it's also in its own separate commit?

Comment (talik077) [reply snyk-operator/deploy/olm-catalog/snyk-operator/0.0.0/snykmonitors.charts.helm.k8s.io.crd.yaml]: is it part of the `chore: improve Operator description`? If not, I would prefer each code change to be in a separate commit. not a blocker tho :)

Comment (ivanstanev) [reply snyk-operator/deploy/olm-catalog/snyk-operator/0.0.0/snykmonitors.charts.helm.k8s.io.crd.yaml]: You've got a point; let's fix this!

**Pull request merged**